release/models/isis/openconfig-isis.yang

```diff
@@ -1397,6 +1422,26 @@ module openconfig-isis {
         packets, overriding the value of the enabled leaf in this context.";
     }
 
+    leaf auth-mode {
```

Comment:
Commenting on behalf of @robshakir

Perhaps make a common grouping that is included in both places here, rather than having the repetition?

Reply:
I have now moved the simple authentication leaves into a separate grouping.